lib/psci/psci_suspend.c

```diff
@@ -170,6 +170,7 @@ void psci_cpu_suspend_start(const entry_point_info_t *ep,
 	psci_acquire_pwr_domain_locks(end_pwrlvl,
 				      idx);
 
+#define ENABLE_EARLY_WAKEUP_DETECTION
```

Comment:
s/define/if/ ?

Reply:
Thanks for catching the typo. Fixed this issue.